src/fdb5/tools/fdb-list.cc

```diff
@@ -93,24 +86,20 @@ void FDBList::init(const CmdArgs& args) {
     porcelain_ = args.getBool("porcelain", false);
     json_ = args.getBool("json", false);
     compact_ = args.getBool("compact", false);
+    depth_     = args.getInt("depth", 3);
```

Comment:
Given that we set the default in the class/constructor, we should default getInt("depth", depth_). That probably applies for the other arguments as well...

Reply:
good point